src/Instance/InstanceDetails/InstanceSubjectView/InstanceSubjectView.js

```diff
@@ -0,0 +1,65 @@
+import React, {
+  useMemo,
+} from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+
+import {
+  Accordion,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import {
+  checkIfArrayIsEmpty,
+} from '../../../utils';
+import {
+  noValue,
+} from '../../../constants';
+
+const visibleColumns = ['Subject headings'];
+const getColumnMapping = intl => ({
+  'Subject headings': intl.formatMessage({ id: 'ui-inventory.subjectHeadings' })
+});
+const classificationsRowFormatter = {
+  'Subject headings': item => item?.value || noValue
+};
+
+const InstanceSubjectView = ({
+  id,
+  subjects,
+}) => {
+  const intl = useIntl();
+
+  const columnMapping = useMemo(() => getColumnMapping(intl), []);
```

Comment:
why it's without `intl` in deps?) looks like eslint doesn't lint your hooks

Reply:
I don't think we need to track `intl` as language can't be changed when this component is mounted

Reply:
it's true that it shouldn't change, but semantically it's better to include this, in case it's changes somehow in a future